src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundleExtractToSpecificPath.cs

```diff
@@ -57,12 +57,20 @@ private void Bundle_Extraction_To_Specific_Path_Succeeds()
             extractDir.Should().NotHaveFiles(BundleHelper.GetFilesNeverExtracted(fixture));
         }
 
-        [InlineData("./foo")]
-        [InlineData("../foo")]
-        [InlineData("foo")]
-        [InlineData("foo/bar")]
+        [InlineData("./foo", BundleOptions.None)]
+        [InlineData("../foo", BundleOptions.None)]
+        [InlineData("foo", BundleOptions.None)]
+        [InlineData("foo/bar", BundleOptions.None)]
+        [InlineData("./foo", BundleOptions.BundleNativeBinaries)]
+        [InlineData("../foo", BundleOptions.BundleNativeBinaries)]
+        [InlineData("foo", BundleOptions.BundleNativeBinaries)]
+        [InlineData("foo/bar", BundleOptions.BundleNativeBinaries)]
+        [InlineData("./foo", BundleOptions.BundleSymbolFiles)]
+        [InlineData("../foo", BundleOptions.BundleSymbolFiles)]
+        [InlineData("foo", BundleOptions.BundleSymbolFiles)]
+        [InlineData("foo/bar", BundleOptions.BundleSymbolFiles)]
         [Theory]
-        private void Bundle_Extraction_To_Relative_Path_Succeeds (string relativePath)
+        private void Bundle_Extraction_To_Relative_Path_Succeeds (string relativePath, BundleOptions bundleOptions)
         {
             var fixture = sharedTestState.TestFixture.Copy();
             var singleFile = BundleSelfContainedApp(fixture, BundleOptions.None);
```

Comment:
The test is not using the `bundleOption` parameter - it always runs with `None`.